pkg/ingester/pyroscope/connect.go

```diff
@@ -0,0 +1,46 @@
+package pyroscope
+
+import (
+	"github.com/bufbuild/connect-go"
+)
+
+func connectCodeToHTTP(code connect.Code) int {
```

Comment:
Should we use `pkg/util/connectgrpc.CodeToHTTP` here?

Reply:
yes!